server_settings-EXAMPLE.py

```diff
@@ -24,7 +24,6 @@
                       'max_team_size': 1,
                       'command_prefix': '/',
                       'include_in_global_lb': False,
-                      'bot_channels': [],  # list of channel IDs that are bot channels
```

Comment:
This should stay, unless I am missing something

Reply:
Check a few lines down, the key is duplicated.